sysutils/glances/Portfile

```diff
@@ -18,17 +18,14 @@ long_description    Glances is a cross-platform monitoring tool which aims to pr
 
 platforms           darwin
 
-checksums           rmd160  300ca1378a1e70cefbd727c490208006a26f3280 \
-                    sha256  eaf8bce4cd4503b7217ae1d825d6dd93ab03d8a66daee402b086b63f73419ed6 \
-                    size 6803273
+checksums           rmd160  434af39cd6a11abb40a6407f14d1e13526a39559 \
+                    sha256  26418380d954d17387bea544b4666f1ee4aefd74d6e41dc08bb3f4b67f390ff1 \
+                    size 6804749
 
-python.default_version 37
-depends_lib         port:py${python.version}-psutil \
-                    port:py${python.version}-future
-
-patch {
-    reinplace "s|/usr/local|${prefix}|" setup.py
-}
+python.default_version 38
+depends_lib-append  port:py${python.version}-psutil \
+                    port:py${python.version}-future \
+                    port:py${python.version}-setuptools
```

Comment:
Is py-setuptools really needed by glances at runtime? Normally it is only needed at build time, and if so, it should be in `depends_build`.

Reply:
That was my call. `py-setuptools` is required at build time for all the usual reasons, but it is also required at runtime since glances is a CLI tool with [entry_points](https://github.com/nicolargo/glances/blob/74a499b5782cfc0a38dd6d3367e02971c273c74a/setup.py#L119).

I find https://github.com/macports/macports-ports/pull/2684#issuecomment-426193304 quite a succinct explanation why, and @reneeotten discusses it in https://github.com/macports/macports-ports/pull/6314#issuecomment-585490282.